probe-rs/targets/STM32WBA_Series.yaml

```diff
@@ -3,27 +3,112 @@ manufacturer:
   id: 0x20
   cc: 0x0
 generated_from_pack: true
-pack_file_release: 1.2.1
+pack_file_release: 1.3.0
 variants:
+- name: STM32WBA50KEUx
+  cores:
+  - name: main
+    type: armv8m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Nvm
+    name: BANK_1
+    range:
+      start: 0x8000000
+      end: 0x8080000
+    cores:
+    - main
+    access:
+      write: false
+      boot: true
+  - !Ram
+    name: SRAM
+    range:
+      start: 0x20000000
+      end: 0x20018000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32wbax_1m_0800
+  - stm32wbax_1m_0c00
+- name: STM32WBA50KGUx
+  cores:
+  - name: main
+    type: armv8m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Nvm
+    name: BANK_1
+    range:
+      start: 0x8000000
+      end: 0x8100000
+    cores:
+    - main
+    access:
+      write: false
+      boot: true
+  - !Ram
+    name: SRAM
+    range:
+      start: 0x20000000
+      end: 0x20020000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32wbax_1m_0800
+  - stm32wbax_1m_0c00
 - name: STM32WBA52CEUx
   cores:
   - name: main
     type: armv8m
     core_access_options: !Arm
-      ap: 1
+      ap: 0
```

Comment:
This is sus, @Dirbaio was `ap: 1` pack data or manual edit?

Reply:
This must have been either manual, or a consequence of an old cmsis-pack/target-gen version, I can't tell.

Reply:
rm0493/figure 490 indicates `ap: 1` is what we want, but why is this not indicated in the packs?

Reply:
yeah this was manual edit. The packs are wrong.

Reply:
Thanks, this at least explains why #2612 does not work.